src/web/mjs/engine/Settings.mjs

```diff
@@ -93,6 +93,13 @@ export default class Settings extends EventTarget {
             value: false
         };
 
+        this.retries = {
+            label: 'Retries',
+            description: 'No of times to retry if the site fetch failes',
```

Comment:
*fails

Reply:
and since the user will be reading this setting it might be worth to not use `no` as shorthand for `number` (non english users ...)